### PR TITLE
Fe/feature/ri 4617 sign in users

### PR DESCRIPTION
**Pull request by arthosofteq**

### CODE REVIEW

File: redisinsight/ui/src/components/promo-link/PromoLink.tsx [hunk 1]
```diff
@@ -28,13 +29,14 @@ const PromoLink = (props: Props) => {
...
     >
+      <EuiIcon type={CloudIcon} size="m" className={styles.cloudIcon} />
       <EuiText className={styles.title}>
         <span style={{ color: linkStyles?.color }}>{title}</span>
       </EuiText>
       <EuiText className={styles.description}>
         <span style={{ color: linkStyles?.color }}>{description}</span>
       </EuiText>
-      {icon && <EuiIcon type={icon} size="m" className={styles.icon} />}
+      {/* {icon && <EuiIcon type={icon} size="m" className={styles.icon} />} */}
```
Comment (romansergeenkosofteq) [inline redisinsight/ui/src/components/promo-link/PromoLink.tsx:39]: please, remove unused code

File: redisinsight/ui/src/electron/components/ConfigElectron/ConfigElectron.tsx [hunk 2]
```diff
@@ -35,6 +53,10 @@ const ConfigElectron = () => {
     }
   }, [isReleaseNotesViewed])
 
+  useEffect(() => {
+    console.log({ currentAccount })
```
Comment (romansergeenkosofteq) [inline redisinsight/ui/src/electron/components/ConfigElectron/ConfigElectron.tsx:57]: console.log

File: redisinsight/ui/src/electron/constants/cloudAuth.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+export enum CloudAuthStatus {
+  Succeed = 'succeed',
+  Failed = 'failed',
+}
+
+export interface CloudAuthResponse {
+  status: CloudAuthStatus
+
+  message?: string
+
```
Comment (romansergeenkosofteq) [inline redisinsight/ui/src/electron/constants/cloudAuth.ts:10]: do we need these empty lines?

File: redisinsight/ui/src/electron/components/ConfigElectron/ConfigElectron.tsx (hunk 2 above)
Comment (zalenskiSofteq) [reply redisinsight/ui/src/electron/components/ConfigElectron/ConfigElectron.tsx:57]: this is needed for testing

Comment (zalenskiSofteq) [reply redisinsight/ui/src/electron/components/ConfigElectron/ConfigElectron.tsx:57]: this is needed for testing

File: redisinsight/ui/src/components/promo-link/PromoLink.tsx (hunk 1 above)
Comment (zalenskiSofteq) [reply redisinsight/ui/src/components/promo-link/PromoLink.tsx]: removed

File: redisinsight/ui/src/electron/constants/cloudAuth.ts (hunk 3 above)
Comment (zalenskiSofteq) [reply redisinsight/ui/src/electron/constants/cloudAuth.ts]: removed

**Review by romansergeenkosofteq — APPROVED** [review]

**Pull request merged**